### ISSUE TITLE
Layer property to always leave it out of prints

### ISSUE THREAD

**Issue by Ellebjerg** [qgis/QGIS#38970]
In my organization we have QGIS users on many levels. To help the newbies and to save time for the rest of us we use both project and composer templates. However, every time a user starts on a new print composition, he or she has to remove a number of raster and vector layers from the legend.

It does not make sense to show the background map in the legend, and since we in our project template have a group of several background maps to chose from, it is not one but several layers that must be removed from the legend each time. In the same way we have a couple of vector layers – municipality border etc. – which it is not necessary to show in the legend.

It would be a great help if one could set up in the properties for a layer in a project that the layer should be left out of the legend in the composer.

This said, I would like to thank all the involved developers for a great product!

Best regards,
Lars Ellebjerg

**Comment by andreasneumann** [comment]
I agree - having a layer property telling QGIS that this layer should be excluded from layout legends, would make life a bit easier.

The question is: should such a layer ALWAYS be excluded or do you expect that there still is a way to include such a layer, if the user really wants it?

**Comment by Ellebjerg** [comment]
If the user wants the layer included he or she should simply change the setting in the layer property in the relevant project. This will not affect other projects based on the same template, so the user is free to do so.

**Comment by NyakudyaA** [comment]
Maybe have an option to allow legend to follow themes and we can encourage people to setup map themes. In this case you would add your layers to a theme and it will be visible in the legend otherwise everything renders
